### PR TITLE
Rewrite PCR & PLS

### PR DESCRIPTION
**Pull request by Pattrigue**
PLS shoot me (XD)

### CODE REVIEW

File: report_thesis/src/sections/background/linear_and_regularization_models/pls.tex [hunk 1]
```diff
@@ -1,46 +1,23 @@
...
+\subsubsection{Principal Component Regression (PCR) \& Partial Least Squares (PLS)}\label{subsec:pls}
 In order to understand \gls{pls}, it is helpful to first consider \gls{pcr}, as \gls{pls} is an extension of \gls{pcr} that aims to address some of its limitations.
+We provide provide an overview of both regression techniques based on \citet{James2023AnIS}.
 
 \gls{pcr} extends \gls{pca} in the context of regression analysis.
-In \gls{pcr}, the dataset $\mathbf{X}$ is decomposed using \gls{pca} as:
+First, \gls{pca} is performed to identify the $k$ principal components that capture the most variance in the data.
+These components are then used in a linear regression model to predict the target variable by fitting a linear model via least squares regression.
+The key intuition behind \gls{pcr} is that a small number of principal components are sufficient to capture most of the variance in the data, which can then be used to predict the target variable.
+That is, the directions of the principal components are assumed to be associated with the target variable.
```
Comment (Ivikhostrup) [inline report_thesis/src/sections/background/linear_and_regularization_models/pls.tex:9]:
```suggestion
That is, the directions of the principal components are assumed to be associated with the target variables.
```

File: report_thesis/src/sections/background/linear_and_regularization_models/pls.tex [hunk 2]
```diff
@@ -1,46 +1,23 @@
...
-The components are then used to predict the target variable by fitting a linear model via least squares regression.
\ No newline at end of file
+Similar to \gls{pcr}, \gls{pls} identifies $k$ components that capture the most variance in the data and fits a linear model with least squares regression.
+However, unlike \gls{pcr}, \gls{pls} uses the target variable to identify the components that are most predictive of the target, resulting in components that not only approximate the data but also relate to the target variable.
+In essence, \gls{pls} aims to find components that are both informative about the data and predictive of the target.
+
+This is an iterative process where the residuals from the previous components are used to calculate the next component.
+The $m$-th component, for example, is derived from the residuals of the previous $m-1$ components.
+One can interpret these residuals as the part of the data that has not been explained by the previous components.
+After some number of iterations, the components are used to predict the target variable by fitting a linear model via least squares regression, just like in \gls{pcr}.
```
Comment (Ivikhostrup) [inline report_thesis/src/sections/background/linear_and_regularization_models/pls.tex:23]: I think a key aspect missing here is that the number of iterations done is equal to the number of components chosen.
In the book they write:
"This iterative approach can be repeated M times to identify multiple PLS components Z1,...,ZM" So you have as many PLS components as you have M.

**Comment by Ivikhostrup** [comment]
I think this section gets to the heart of PLS without too much detail. 
I have one "large" correction regarding the number of components but thats it.

File: report_thesis/src/sections/background/linear_and_regularization_models/pls.tex (hunk 1 above)
Comment (Pattrigue) [reply report_thesis/src/sections/background/linear_and_regularization_models/pls.tex:9]: Disagree. "Target variable" is correct if predicting a single outcome, as is typical in Principal Component Regression, while "target variables" would imply multiple outcomes.

File: report_thesis/src/sections/background/linear_and_regularization_models/pls.tex [hunk 3]
```diff
@@ -1,46 +1,23 @@
...
-$$
-
-The components are then used to predict the target variable by fitting a linear model via least squares regression.
\ No newline at end of file
+Similar to \gls{pcr}, \gls{pls} identifies $k$ components that capture the most variance in the data and fits a linear model with least squares regression.
+However, unlike \gls{pcr}, \gls{pls} uses the target variable to identify the components that are most predictive of the target, resulting in components that not only approximate the data but also relate to the target variable.
+In essence, \gls{pls} aims to find components that are both informative about the data and predictive of the target.
+
+This is an iterative process where the residuals from the previous components are used to calculate the next component.
+The $m$-th component, for example, is derived from the residuals of the previous $m-1$ components.
```
[the remark this answers is not in the document]
Comment (Pattrigue) [reply report_thesis/src/sections/background/linear_and_regularization_models/pls.tex:22]:
```suggestion
That is, the $m$-th component, for example, is derived from the residuals of the previous $m-1$ components.
```

File: report_thesis/src/sections/background/linear_and_regularization_models/pls.tex (hunk 2 above)
Comment (Pattrigue) [reply report_thesis/src/sections/background/linear_and_regularization_models/pls.tex]: Should be fixed now.

**Review by Ivikhostrup — APPROVED** [review]

**Review by chhoumann — APPROVED** [review]

**Pull request merged**